app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
```

Comment:
fix imports ordering 
![Screenshot from 2023-10-13 13-38-45](https://github.com/mate-academy/py-cat-and-dog-years/assets/122517203/3a77a593-5700-4fa8-a038-63b5b2cc71d3)